openapi2kong/validator.go

```diff
@@ -98,7 +108,11 @@ func generateBodySchema(operation *openapi3.Operation) string {
 	}
 
 	for contentType, content := range content {
-		if strings.Contains(strings.ToLower(contentType), "application/json") {
+		typ, subtype, err := parseMediaType(contentType)
+		if err != nil {
+			panic(err)
```

Comment:
do not panic here, this is a library and it shouldn't bail out but return the error.

Imho 2 options:

1. use `logbasics` and log a message at `Info` level that the mediatype found is invalid and will be ignored
2. bubble up the error to the caller.

Reply:
fyi in the entire library, only the `Must...` methods/functions will ever panic. everything else will return an error.

Reply:
This sounds good to me. I print an Info log when the mediatype failed to be parsed